lib/l10n/app_en.arb

```diff
@@ -2,7 +2,7 @@
     "@@locale": "en",
     "appTitle": "What to expect from Ubuntu on WSL",
     "windowTitle": "Installing Ubuntu on WSL",
-    "welcome": "Welcome to Ubuntu",
+    "welcome": "Welcome to Ubunt WSL",
```

Comment:
oops, missing "u" :smile:

Reply:
It's surpising how we become blind to those things... Thanks for spotting that out.